src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/pages/Login.vue

```diff
@@ -34,7 +34,9 @@
               <v-btn rounded="lg" class="font-weight-bold" color="primary" @click="handleLogin('bcsc')">LOG IN WITH BC SERVICES CARD</v-btn>
               <div class="text-center">
                 <p class="small">Don't have an account?</p>
-                <a href="https://id.gov.bc.ca/account/setup-instruction"><p class="small">Get set up with BC Services Card</p></a>
+                <a href="https://id.gov.bc.ca/account/setup-instruction">
+                  <p class="small">Register for BC Services Card</p>
+                </a>
```

Comment:
I think this was a requested copy change to 

"Get set up with BC Services Card"